### ISSUE TITLE
Instant loading doesn't work well when jumping to local .html file

### ISSUE THREAD

**Issue by Mi11et** [squidfunk/mkdocs-material#1653]
__I checked that...__

- [x] ... the documentation does not mention anything about my problem
- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... there are no open or closed issues that are related to my problem

## Description

When clicking a link to another local .html file (which is not generated by MkDocs) with instant feature on, the page won't load correctly.

### Expected behavior

Instant loading should ignore that link.

### Actual behavior

The page doesn't load correctly.

### Steps to reproduce the bug

1. Create a new MkDocs project.
``` shell
mkdocs new test
```
2. Set theme to "material" and enable "instant" feature.
``` yaml
theme:
  name: material
  features:
    - instant
```
3. Create `test.html`.
``` html
<html>
    <body>
        <p>
            test
        </p>
    </body>
</html>
```
4. Write a link to that page in `index.md`.
``` markdown
[test](/test.html)
```
5. Run `mkdocs serve` and check that link.

### Package versions

* Python: `3.7.2`
* MkDocs: `1.1`
* Material: `5.1.0`

### Project configuration

``` yaml
site_name: Test

theme:
  name: material
  features:
    - instant
```

### System information

* OS: Windows 10
* Browser: I've tried both Firefox and Chrome

**Comment by squidfunk** [comment]
Just a quick idea for later evaluation: when instant loading is executed and the response is received, we could check for the `generator` meta tag (or maybe others) and perform a full page load, if not found.

**Comment by squidfunk** [comment]
Potential fix in #1668. Note that there are currently some limitations when serving the docs locally, but it should fully work when deployed. Before merging and releasing, we need to collect some feedback.

**Comment by squidfunk** [comment]
No feedback, so I guess I’ll just merge the stuff whenever I find some time and we see if it breaks something after releasing it.

**Comment by squidfunk** [comment]
Released as part of 5.1.6
